cadc-data-ops-fits/build.gradle

```diff
@@ -14,13 +14,18 @@ sourceCompatibility = 1.8
 group = 'org.opencadc'
 version = '0.1.2'
 
+// To allow for french extended ASCII characters in file javadoc.
+javadoc {
```

Comment:
We've never seen a need for this before... are the offending characters from the french in the copyright header? Is this another thing caused by gradle 7 or does it also effect CI with gradlew and gradle 6.8.3? 

We have so many small modules that I don't like to solve problems with something in every build file if I can possibly avoid it... please remove it here and create an issue instead.